screen/model/commandrouter.js

```diff
@@ -0,0 +1,59 @@
+
+let MediaHandler = require("./media/mediahandler");
```

Comment:
If you don't reassign the variable later, `const` is better than `let` for peace of mind.